### PR TITLE
add new coverage simplifier plugin from new JTS coverage code

### PR DESCRIPTION
**Pull request by mukoki**
This plugin takes advantage of a new capability of JTS available in the coverage package.

### CODE REVIEW

File: src/language/jump_zh_CN.properties [hunk 1]
```diff
@@ -2114,10 +2114,10 @@ ui.plugin.OpenProjectPlugIn.open-project = 打开项目
...
 ui.plugin.OptionsPlugIn.snap-vertices-tools = 顶点对齐工具
 ui.plugin.OptionsPlugIn.view-edit = 查看/编辑
-ui.plugin.PasteItemsPlugIn.clipboard-must-not-be-empty = \#T\:
-ui.plugin.PasteItemsPlugIn.clipboard-must-contain-geometries-or-wkt = \#T\:
-ui.plugin.PasteLayersPlugIn.clipboard-must-not-be-empty = \#T\:
-ui.plugin.PasteLayersPlugIn.clipboard-must-contain-layers = \#T\:
+ui.plugin.PasteItemsPlugIn.clipboard-must-not-be-empty=#T:
+ui.plugin.PasteItemsPlugIn.clipboard-must-contain-geometries-or-wkt=#T:
+ui.plugin.PasteLayersPlugIn.clipboard-must-not-be-empty=#T:
+ui.plugin.PasteLayersPlugIn.clipboard-must-contain-layers=#T:
```
Comment (edeso) [inline src/language/jump_zh_CN.properties:2120]: unrelated. how about we remove translations that were not maintained for years.

i expect the quality of those to be underwhelming at least.

**Comment by edeso** [comment]
did a quick test and works as expected. no objections here.

Q: how about a new release just in time for the next OSGeoLive?

**Comment by jratike80** [comment]
> Q: how about a new release just in time for the next OSGeoLive?

Oh yes, this plugin alone would prove that OJ is not dead and it is ahead for example GDAL https://github.com/OSGeo/gdal/issues/11540

**Comment by ma15569** [comment]
Hi,
I am ok for the OSGeoLive.

Il giorno gio 26 dic 2024 alle ore 16:28 Jukka Rahkonen <
***@***.***> ha scritto:

**Comment by mukoki** [comment]
Just merged the PR. Thank you for the test.
Ok for a release for OSGeoLive.
Also OK for removing un-maintained translation files after the release (maybe we can keep them in etc or doc folder : files with more than 1000 missing translations over 3000 are ja, ml, pt, pt_BR, ta, zh_CN, zh_HK).

**Comment by ma15569** [comment]
Ok for me to remove. I

Il lun 30 dic 2024, 00:32 Michaël Michaud ***@***.***> ha
scritto:

**Comment by edeso** [comment]
> Also OK for removing un-maintained translation files after the release (maybe we can keep them in etc or doc folder : files with more than 1000 missing translations over 3000 are ja, ml, pt, pt_BR, ta, zh_CN, zh_HK).

hmm, i'd rather go for didn't receive any contribution in so or so many years? how'd you compute the missing translation counts? shell grep for '#T:'?

**Comment by mukoki** [comment]
Hey, sorry for late answer,Yes, I counted the #T:. Not very precise, but it is difficult to identify last contributions as files are updates (with new #T:) even when there are no new translation.Anyway, I'm OK with any other way to evaluate unmaintained I18N files.Michaelenvoyé : 30 décembre 2024 à 16:26de : edeso ***@***.***>à : openjump-gis/openjump ***@***.***>cc : Michaël Michaud ***@***.***>, State change ***@***.***>objet : Re: [openjump-gis/openjump] add new coverage simplifier plugin from new JTS coverage code (PR #142) Also OK for removing un-maintained translation files after the release (maybe we can keep them in etc or doc folder : files with more than 1000 missing translations over 3000 are ja, ml, pt, pt_BR, ta, zh_CN, zh_HK).hmm, i'd rather go for didn't receive any contribution in so or so many years? how'd you compute the missing translation counts? shell grep for '#T:'?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you modified the open/close state.Message ID: ***@***.***>

**Pull request merged**